src/Model/index.d.ts

```diff
@@ -53,6 +53,7 @@ declare module '@nozbe/watermelondb/Model' {
 
     public observe(): Observable<this>
 
+    // @ts-ignore
```

Comment:
This was necessary to get TypeScript not to complain. Could add more descriptive comment if needed.